pkg/controller/disruption/disruption.go

```diff
@@ -359,6 +359,8 @@ func verifyGroupKind(controllerRef *metav1.OwnerReference, expectedKind string,
 func (dc *DisruptionController) Run(ctx context.Context) {
 	defer utilruntime.HandleCrash()
 	defer dc.queue.ShutDown()
+	defer dc.recheckQueue.ShutDown()
+	defer dc.broadcaster.Shutdown()
```

Comment:
The problem is that this things panic right now :/

https://github.com/kubernetes/kubernetes/pull/108080

Reply:
Should be fixed now.